pkg/bootstrap/testdata/all_golden.json

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
nit: please name the file based on the config used to generate it.. bootstrap-with-auth.json.golden